Lib/fontTools/ttLib/tables/otTables.py

```diff
@@ -927,9 +926,10 @@ def postRead(self, rawTable, font):
 				cls = rec.Class
 				startID = font.getGlyphID(start)
 				endID = font.getGlyphID(end) + 1
-				for glyphID in range(startID, endID):
-					if cls:
-						classDefs[font.getGlyphName(glyphID)] = cls
+				glyphNames = font.getGlyphNameMany(range(startID, endID))
+				if cls:
```

Comment:
If `not cls`, then `glyphNames` is unused, so `glyphNames` should only be defined within this block.